sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -112,15 +112,15 @@ export class StudentController extends BaseController {
   async create(
     @Body() payload: CreateStudentDto,
     @UserToken() userToken: IUserToken,
-  ) {
+  ): Promise<void> {
```

Comment:
Me and Signori discussed this and unless it is needed, there is no point in sending the student dto back. If we wanted to, because they were being used in Client like so, we need to figure out which fields are needed. At this moment, sending a 201 CREATED is good enough. Let me know if devs need to discuss this

Reply:
@fwpushan, my understanding is that POST doesn't need to return anything else that, maybe, an identifier that will allow the API consumer to have access to the newly created resource. We should avoid return unnecessary data.
Previously this method was returning the model from DB. We can consider this change as an improvement in comparison with the previous way.

Reply:
OK, Normally rest API specify a [location header for newly created resources](https://restfulapi.net/http-status-201-created/), if no such endpoint exists, normally people send newly created object. But if you, people think that is unnecessary, I'm ok with that.

Reply:
Thanks.